src/authorization/Authorizations.ts

```diff
@@ -725,6 +724,10 @@ export default class Authorizations {
     return user.role === UserRole.DEMO;
   }
 
+  public static isTechnical(user: UserToken | User): boolean {
```

Comment:
No need to create a method to check a boolean ;-)
Just use user.technical in the IF statements.

Reply:
done